.travis.yml

```diff
@@ -1,58 +1,58 @@
 language: node_js
-
+version: 1.0
 node_js:
-  - "node"
-  - "lts/*"
-  
+- node
+- lts/*
 before_script:
-  - REPO_VERSION="$(npx -c 'echo "$npm_package_version"')"
-  - echo "REPO_VERSION=$REPO_VERSION"
-  - npm run build
-  - TPLANT_VERSION="$(node dist/index.js -V)"
-  - echo "TPLANT_VERSION=$TPLANT_VERSION"
-  - NPM_PACKAGE_VERSION="$(npm view tplant version)"
-  - echo "NPM_PACKAGE_VERSION=$NPM_PACKAGE_VERSION"
-  - echo "TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST"
-  - echo "TRAVIS_BRANCH=$TRAVIS_BRANCH"
-  - |
-    if [ "$REPO_VERSION" != "$TPLANT_VERSION" ]; then
-      echo -e '\033[0;31mPackage version and commander version must be the same.';
-      travis_terminate 1;
-    fi
-  - |
-    if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_BRANCH" == "master" ] && [ "$REPO_VERSION" == "$NPM_PACKAGE_VERSION" ]; then
-      echo -e '\033[0;31mPackage version must be higher than the current one.';
-      travis_terminate 1;
-    fi
-  - npm run lint
-
+- REPO_VERSION="$(npx -c 'echo "$npm_package_version"')"
+- echo "REPO_VERSION=$REPO_VERSION"
+- npm run build
+- TPLANT_VERSION="$(node dist/index.js -V)"
+- echo "TPLANT_VERSION=$TPLANT_VERSION"
+- NPM_PACKAGE_VERSION="$(npm view @jeff-tian/tplant version)" || NPM_PACKAGE_VERSION=0
+- echo "NPM_PACKAGE_VERSION=$NPM_PACKAGE_VERSION"
+- echo "TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST"
+- echo "TRAVIS_BRANCH=$TRAVIS_BRANCH"
+- |
+  if [ "$REPO_VERSION" != "$TPLANT_VERSION" ]; then
+    echo -e '\033[0;31mPackage version and commander version must be the same.';
+    travis_terminate 1;
+  fi
+- |
+  if [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_BRANCH" == "fork" ] && [ "$REPO_VERSION" == "$NPM_PACKAGE_VERSION" ]; then
```

Comment:
I would keep this as `master`

Reply:
It's mis-operation. I just need the feature yesterday so published one to @jeff-tian/tplant. Reverted it.